eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/ContigReplacerProcessorTest.java

```diff
@@ -38,61 +40,132 @@
 
     private static final String ALTERNATE_ALLELE = "T";
 
+    private static final String CHROMOSOME = "1";
+
+    private static final String MISSING_CHROMOSOME = "23";
+
+    private static final String SEQNAME_1 = "22";
+
+    private static final String GENBANK_CONTIG = "CM000994.2";
+
     private static final String GENBANK_1 = "genbank_example_1";
 
     private static final String REFSEQ_1 = "refseq_example_1";
 
-    private static final String SEQNAME_1 = "22";
+    private static final String MISSING_CONTIG = "NT_missing";
 
     private static final String UCSC_1 = "ucsc_example_1";
 
     private static final Long SS_ID = 12345L;
 
     private static final Long RS_ID = 56789L;
 
+    private static final String REFSEQ_WITHOUT_SYNONYM = "NT_without_synonym";
+
+    private static final String GENBANK_WITHOUT_SYNONYM = "GL_without_synonym";
+
+    private static final String SEQNAME_WITHOUT_SYNONYM = "CHR_without_synonym";
+
+    private static final String REFSEQ_ASSEMBLY_ACCESSION = "GCF_000001635.26";
+
+    private static final String GENBANK_ASSEMBLY_ACCESSION = "GCA_000001635.8";
+
     private ContigReplacerProcessor processor;
 
+    private ContigReplacerProcessor genbankProcessor;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
     @Before
     public void setUp() throws Exception {
         String fileString = ContigMappingTest.class.getResource("/input-files/assembly-report/AssemblyReport.txt").toString();
         ContigMapping contigMapping = new ContigMapping(fileString);
```

Comment:
I just noticed we don't have tests to check that at least the number of records in a `ContigMapping` is right after loading the assembly report.